src/main/java/apoc/export/cypher/MultiStatementCypherSubGraphExporter.java

```diff
@@ -31,11 +31,13 @@
     Set<String> indexedProperties = new LinkedHashSet<>();
     private long artificialUniques = 0;
     private ExportFormat exportFormat = ExportFormat.NEO4J_SHELL;
+    private boolean merge = false;
```

Comment:
I don't like all the conditionals in all these places. I would prefer a formatter, with "formatNode" and formatRelationship methods that we have two implementations for one for CREATE one for MERGE and perhaps others, that use MATCH or MERGE for the nodes of a relationship.